silverorange/site
# JWPlayer 7.12

## How to test
 - In your site ini, use the production values for `cdn.distribution`, `cdn.private_distribution`, `uri.cdn_base` and `uri.secure_cdn_base`
 - The ini setting for the `jwplayer.key` needs to be updated. See  https://github.com/silverorange/course-host/pull/746

## Tested:

**Loading video**:
Chrome (Windows, OS X), Safari, Firefox (Windows, OS X), iOS Chrome, iOS Safari, IE 11, Android (5.0, 6.0, 7.0)

**Seeking**:
Chrome (Windows, OS X), Safari, Firefox (Windows, OS X), iOS Chrome, iOS Safari, IE 11, Android (5.0, 6.0, 7.0)

**Resume Overlay**:
Chrome (Windows, OS X), Safari, Firefox (Windows, OS X), iOS Chrome, iOS Safari, IE 11, Android (5.0, 6.0, 7.0)

**Variable Speed Controls**:
Chrome (Windows, OS X), Safari, Firefox (Windows, OS X), iOS Chrome, iOS Safari, IE 11, Android (not supported)

**With Flash**:
Chrome (Windows, OS X), Firefox (Windows, OS X), IE 11

**Without Flash**:
Chrome (Windows, OS X), Firefox (Windows, OS X), Safari, iOS Chrome, iOS Safari, IE 11

**HLS**:
Chrome (Windows, OS X), Safari, Firefox (Windows, OS X), iOS Chrome, iOS Safari, IE 11, Android (5.0, 6.0, 7.0)

**MP4**:
Chrome (Windows, OS X), Safari, Firefox  (Windows, OS X)

## Issues:
 - ~~sort out CORS issue~~
 - ~~Safari shows a CC icon~~
 - ~~test in Firefox, Android browsers~~
 - update all site ini files to use the new API key from JW Player
 - ~~update all S3 CORS configs~~
